### DISCUSSION TITLE
Reflexiones y reinterpretación del 🌎 y el 🐈

### DISCUSSION THREAD

**Discussion by mmasias** [mmasias/IdSw2-22-23#9] [Announcements]
De acuerdo a lo visto en la clase de hoy, les dejo abierto el [repo de theWorld](https://github.com/mmasias/theWorld) con mi interpretación del mundo y el gato que se mueve sobre él, parte de lo que sería la solución del [reto de la aspiradora](https://github.com/mmasias/IdSw2-22-23/blob/main/docs/proyectos/pyAspiradora.md). 

Como la solución que aquí se genera causó cierta polémica, la someto a su consideración: la hacemos evaluación continua individual y además, entre todos los que envíen una propuesta && contesten a este mensaje con "Recibido" nos jugamos una 🍕 que no pagará el que envíe la solución que elijamos como la más elegante, teniendo en cuenta los criterios vistos en clase.

Fecha límite para enviar propuestas, la siguiente clase.

Espacio para preguntas y debate, este hilo de discusión.

¡Saludos y seguimos en contacto!

**Comment by mmasias** [comment]
> De acuerdo a lo visto en la clase de hoy, les dejo abierto el [repo de theWorld](https://github.com/mmasias/theWorld) con mi interpretación del mundo y el gato que se mueve sobre él, parte de lo que sería la solución del [reto de la aspiradora](https://github.com/mmasias/IdSw2-22-23/blob/main/docs/proyectos/pyAspiradora.md).
> 
> Como la solución que aquí se genera causó cierta polémica, la someto a su consideración: la hacemos evaluación continua individual y además, entre todos los que envíen una propuesta && contesten a este mensaje con "Recibido" nos jugamos una 🍕 que no pagará el que envíe la solución que elijamos como la más elegante, teniendo en cuenta los criterios vistos en clase.
> 
> Fecha límite para enviar propuestas, la siguiente clase.
> 
> Espacio para preguntas y debate, este hilo de discusión.
> 
> ¡Saludos y seguimos en contacto!

La propuesta debe consistir en un modelo nuevo que plantee las clases que permitan que exista el mundo, la suciedad y el gato, así como su implementación en la que se vea la correspondencia exquisita entre el modelo y lo programado.

Let's code!

**Comment by ManPuyol** [comment]
> De acuerdo a lo visto en la clase de hoy, les dejo abierto el [repo de theWorld](https://github.com/mmasias/theWorld) con mi interpretación del mundo y el gato que se mueve sobre él, parte de lo que sería la solución del [reto de la aspiradora](https://github.com/mmasias/IdSw2-22-23/blob/main/docs/proyectos/pyAspiradora.md).
> 
> Como la solución que aquí se genera causó cierta polémica, la someto a su consideración: la hacemos evaluación continua individual y además, entre todos los que envíen una propuesta && contesten a este mensaje con "Recibido" nos jugamos una 🍕 que no pagará el que envíe la solución que elijamos como la más elegante, teniendo en cuenta los criterios vistos en clase.
> 
> Fecha límite para enviar propuestas, la siguiente clase.
> 
> Espacio para preguntas y debate, este hilo de discusión.
> 
> ¡Saludos y seguimos en contacto!

Recibido

[Repositorio con mi versión](https://github.com/ManPuyol/theWorld/tree/ortega)
